### ISSUE TITLE
Cant repair buildings that are powered down. [Playtest-20151129]

### ISSUE THREAD

**Issue by epicelite** [OpenRA/OpenRA#10187]
If you use the little electricity button to turn off a building then you click the wrench to repair it, it wont repair until you give it power.
Unless this is intentional for some reason?

**Comment by UberWaffe** [comment]
I've attempted to reproduce this in bleed without any success. (Have NOT tested this in playtest 20151129).

Tested in RA, using power-down and repair on a tesla coil. Tried all manner of order.
First power down, then start repairs, then toggle repair on and off, and power on and off.
First start repair, then toggle power off, then toggle repair and power on and off.
Tried with both base in lower power mode, and sufficient power mode, and edge case where power off and on the tesla coils switches between low-power state or not.

So it at least appears to work fine on bleed.
